### ISSUE TITLE
Automatic scrolling on directory page

### ISSUE THREAD

**Issue by RebeccaOFN** [openfoodfoundation/sib-discovery-components#78]
Hi @mkllnk , one more issue I'm seeing is that when you view the directory page, it automatically loads/scrolls down to where the directory code is inserted, rather than at the top of the page, so to see the header you would have to scroll back up. You can see what I mean here: https://waterlooregionfood.ca/staging/5285/directory/. Did you have this issue at all on the Discover Regenerative site or have any idea how to fix it?

**Comment by mkllnk** [comment]
Yes, we have the same issue. It's on our list for SiB to fix but we are not quite there yet. Do you have budget to get that fixed?
